File: 2_course/ЯПВУ/Laptev_AV1/Лаптев Александр/LaptevAV-1_13.go
package main
import ("fmt"
        "math"
)

func main() {
    var x1 float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&x1)
    var y1 float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&y1)
    var x2 float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&x2)
    var y2 float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&y2)
    var x3 float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&x3)
    var y3 float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&y3)
    s := math.Abs (float64((x1 - x3) * (y2 - y3) - ((y1 - y3) * (x2 - x3)))) / 2
    fmt.Println (s)
}

File: 2_course/ЯПВУ/Laptev_AV1/Лаптев Александр/LaptevAV-1_13.rb
# encoding utf-8
puts ('Vvedite veshestvennoe chislo: ')
x1 = gets.to_i
puts ('Vvedite veshestvennoe chislo: ')
y1 = gets.to_i
puts ('Vvedite veshestvennoe chislo: ')
x2 = gets.to_i
puts ('Vvedite veshestvennoe chislo: ')
y2 = gets.to_i
puts ('Vvedite veshestvennoe chislo: ')
x3 = gets.to_i
puts ('Vvedite veshestvennoe chislo: ')
y3 = gets.to_i
s = ((x1 - x3) * (y2 - y3) - ((y1 - y3) * (x2 - x3))) / 2
print ("#{s.abs}")